### PR TITLE
add support unregister alias

### PR DESCRIPTION
**Pull request by yanyandenuonuo**
Once set alias for some real key, there is no way to unset the alias or override the alias.

### CODE REVIEW

**Comment by yanyandenuonuo** [comment]
The check failed is not my faults, please check the ci because it can't download some package before build.

**Comment by sagikazarmark** [comment]
@yanyandenuonuo What exactly is your use case for unregistering an alias?

If you rebase from the latest master, the builds will be fixed.

File: viper.go [hunk 1]
```diff
@@ -1143,14 +1143,30 @@ func (v *Viper) registerAlias(alias string, key string) {
...
-		jww.DEBUG.Println("Alias", key, "to", newkey)
-		return v.realKey(newkey)
+		jww.DEBUG.Println("Alias", key, "to", newKey)
+		return v.realKey(newKey)
 	}
 	return key
 }
 
+// Remove the alias.
+// This enables one to remove a alias and then override the alias
```
Comment (sagikazarmark) [inline viper.go]:
```suggestion
// This enables one to remove a registered alias.
```

File: viper.go [hunk 2]
```diff
@@ -1143,14 +1143,30 @@ func (v *Viper) registerAlias(alias string, key string) {
...
 	if exists {
-		jww.DEBUG.Println("Alias", key, "to", newkey)
-		return v.realKey(newkey)
+		jww.DEBUG.Println("Alias", key, "to", newKey)
+		return v.realKey(newKey)
 	}
 	return key
 }
 
+// Remove the alias.
```
Comment (sagikazarmark) [inline viper.go]:
```suggestion
```

File: viper.go [hunk 3]
```diff
@@ -1143,14 +1143,30 @@ func (v *Viper) registerAlias(alias string, key string) {
...
+func (v *Viper) UnregisterAlias(alias string) {
+	v.unregisterAlias(alias)
+}
+
+func (v *Viper) unregisterAlias(alias string) {
+	alias = strings.ToLower(alias)
+	if _, exists := v.aliases[alias]; exists {
+		delete(v.aliases, alias)
+	} else {
+		jww.DEBUG.Println("Alias already unregister alias", alias)
```
Comment (sagikazarmark) [inline viper.go]:
```suggestion
		jww.DEBUG.Println("Trying to unregister a non-existent alias", alias)
```

**Pull request closed without merging**